app/views/idv/in_person/address.html.erb

```diff
@@ -1,15 +1,92 @@
+  <%= validated_form_for(
+        :doc_auth, url: url_for, method: 'PUT',
+                   html: { autocomplete: 'off', class: 'margin-top-2' }
+      ) do |f| %>
+    <div class="margin-bottom-3">
+      <%= render ValidatedFieldComponent.new(
+            name: :address1,
+            form: f,
+            label: t('in_person_proofing.form.address.address1'),
+            required: true,
+            maxlength: 255,
+            input_html: { aria: { invalid: false }, value: pii['address1'] },
+          ) %>
+    </div>
+    <div class="margin-bottom-3">
+      <%= render ValidatedFieldComponent.new(
+            name: :address2,
+            form: f,
+            label: t('in_person_proofing.form.address.address2'),
+            required: false,
+            maxlength: 255,
+            input_html: { value: pii['address2'] },
+          ) %>
+    </div>
+    <div class="margin-bottom-3">
+      <%= render ValidatedFieldComponent.new(
+            name: :city,
+            form: f,
+            label: t('in_person_proofing.form.address.city'),
+            required: true,
+            maxlength: 255,
+            input_html: { aria: { invalid: false }, class: 'usa-input', value: pii['city'] },
+          ) %>
+    </div>
+
+    <%= render ValidatedFieldComponent.new(
+          name: :state,
+          form: f,
+          collection: us_states_territories,
+          label: t('in_person_proofing.form.address.state'),
+          prompt: t('in_person_proofing.form.address.state_prompt'),
+          required: true,
+          selected: pii['state'],
+        ) %>
+
+    <div class="tablet:grid-col-6 margin-bottom-3">
+      <%# using :tel for mobile numeric keypad %>
+      <%= render ValidatedFieldComponent.new(
+            name: :zipcode,
+            form: f,
+            as: :tel,
+            label: t('in_person_proofing.form.address.zipcode'),
+            required: true,
+            pattern: '(\d{5}([\-]\d{4})?)',
+            input_html: { aria: { invalid: false }, class: 'usa-input', value: pii['zipcode'] },
+            error_messages: { patternMismatch: t('idv.errors.pattern_mismatch.zipcode') },
+          ) %>
+    </div>
+
+
+    <div class="margin-top-0">
+      <%= render ValidatedFieldComponent.new(
+            name: :same_address_as_id,
+            form: f,
+            as: :radio_buttons,
+            wrapper: :borderless_uswds_radio_buttons,
+            label: t('in_person_proofing.form.address.same_address'),
+            required: true,
+            collection: [
+              [t('in_person_proofing.form.address.same_address_choice_yes'), true],
+              [t('in_person_proofing.form.address.same_address_choice_no'), false],
+            ],
+            input_html: { value: pii['same_address_as_id'] },
+          ) %>
+      </div>
+    <button type="submit" class="usa-button usa-button--big usa-button--wide margin-top-1">
+      <%= t('forms.buttons.continue') %>
+    </button>
```

Comment:
Maybe not an especially compelling use-case for it, but there is a ButtonComponent to help assist with the implementation details of the modifiers you're after:

```suggestion
    <%= render ButtonComponent.new(big: true, wide: true, class: 'margin-top-1') do %>
      <%= t('forms.buttons.continue') %>
    <% end %>
```

Reply:
Done in https://github.com/18F/identity-idp/commit/293fd22edcee96b6208a130183028d1d15b6f71e